### DISCUSSION TITLE
NoteButton error

### DISCUSSION THREAD

**Discussion by klausnielsen** [tttapa/Control-Surface#1025] [General]
Hi everyone,
I'm getting a weird error when trying to use the examples provided on [NoteButton](https://tttapa.github.io/Control-Surface-doc/Doxygen/de/dcc/NoteButton_8ino-example.html) and [Custom-Midi-Output-Element](https://tttapa.github.io/Control-Surface-doc/Doxygen/d4/d3b/Custom-MIDI-Output-Element_8ino-example.html). I simply copy the code, make no changes, and the IDE gives me this error message:
`Compilation error: no match for 'operator[]' (operand types are 'const cs::MIDI_Notes::Note' and 'int')` 
... pointing to this bit in the code:
```C
MyNoteButton button {
  5,                             // Push button on pin 5
  {MIDI_Notes::C[4], Channel_1}, // Note C4 on MIDI channel 1
  0x7F,                          // Maximum velocity
};
```
Any ideas?
I have other sketches using the library that work fine.
Cheers
Klaus

**Comment by tttapa** [comment]
Are you using the latest version of the library? The examples you linked to are for the latest `main` branch.

**Reply by klausnielsen** [reply]
Ahh - perfect! Updating did the trick. Thanks Pieter :)
